src/app.rs

```diff
@@ -2,24 +2,23 @@ use crate::args::Args;
 use crate::colorscheme::Colorscheme;
 use crate::widgets::*;
 
-pub struct App<'a, 'b> {
```

Comment:
Removing that would cause problems in case another patch was to be applied that depends on them. Consider rewriting this part.